tests/unit/test_coords.py

```diff
@@ -1,29 +0,0 @@
-import neurodamus.metype as metype
```

Comment:
commected to the comment before: I think the function is used

Reply:
re-added.